### ISSUE TITLE
Heltec WiFi LoRa V3.1 - Starting LoRa failed!

### ISSUE THREAD

**Issue by erikdewildt** [richonguzman/LoRa_APRS_iGate#98]
Hi, 

I've tried to flash a Heltec WiFi LoRa 32 V3 board (board says version 3.1) with the APRS IGate firmware.

When te board boots it immediately outputs a "Starting LoRa failed!" message and nothing more happens.

The screen shows a "LoRa APRS < IGATE & DIGI >" screen but further I can only reset the board.

Firmware on the screen shows 2024.04.13

Logs:

> ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0x1 (POWERON),boot:0x8 (SPI_FAST_FLASH_BOOT)
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x44c
load:0x403c9700,len:0xbe4
load:0x403cc700,len:0x2a38
entry 0x403c98d4
E (228) SPIFFS: mount failed, -10025
[   228][E][SPIFFS.cpp:89] begin(): Mounting S
Starting Station:    Version: 2024.04.13
Starting LoRa failed!

Any help would be greatly appreciated.
Thanks,

73 de PE1PUX
Erik

**Comment by richonguzman** [comment]
Hi , if you flashed this with the webInstaller (2024-04-13) , this is a known bug as the installer was created with ESP32 and not ESP32S3 as it should.

You can install it the "old way" with VSCODE to have the HELTEC V3 work fine and also have the latest improvements which are a lot (2024-05-17)

Will try to update the web installer soon , but in the meantime please do it over VSCODE

**Comment by erikdewildt** [comment]
Hi, 

I've noticed the issue with the Web Flasher and started using VSCODE to flash the board. This resulted in the "LoRa Failed" error. However after a couple of more tries I got it working. When changing some settings I got the "LoRa failed" error again.

Re-did the complete flash using VSCODE and then was able to configure as desired. So I'm not sure what caused the error.

I'll close this ticket for now as it seems to work.

Thanks,
73

Erik
